config/common-packages.yml

```diff
@@ -18,13 +18,23 @@ install:
   - chkrootkit
   - koji
   - bubblewrap-suid
-  
+
 remove:
   - firefox
   - firefox-langpacks
   - virtualbox-guest-additions
   - openssh-server
   - bubblewrap
+  - cups
+  - gutenprint-cups
+  - cups-browsed
+  - bluez-cups
+  - cups-filters
+  - braille-printer-app
+  - toolbox
```

Comment:
these are separated out for a reason and putting them here causes build failures. Again, do not submit PRs until they are ready. If your build is failing, it means it wasn't ready and you didn't test it before submitting the PR.